src/locales/en.json

```diff
@@ -579,22 +625,22 @@
     "title": "No rates found"
   },
   "ocp_cloud_dashboard": {
-    "compute_title": "Cloud infrastructure compute services usage",
+    "compute_title": "Compute services usage",
     "compute_trend_title": "Daily usage comparison ({{units}})",
     "cost_label": "Cost",
-    "cost_title": "OpenShift cloud infrastructure cost",
-    "cost_trend_title": "Cloud infrastructure cumulative cost comparison ({{units}})",
+    "cost_title": "All infrastructure filtered by OpenShift cost",
+    "cost_trend_title": "All OpenShift related infrastructure cumulative cost comparison ({{units}})",
```

Comment:
Just curious. Why would you have the ned for the 'All' and 'filtered by' here? It just seems strange to have search and filter key words in the title. Nothing to hold up the PR.

Reply:
Good question. The short answer is; this matches Natalie's mocks. We may need to wordsmith this a bit...

I believe "all" is used to convey that this includes OpenShift, Aws, and Azure.

Mock:
<img width="1230" alt="Screen Shot 2020-02-28 at 11 44 04 AM" src="https://user-images.githubusercontent.com/17481322/75567519-ad86cc80-5a1f-11ea-8b4d-72dcd459ab2f.png">

Reply:
Good point. If the data set is made up of a culmination of providers then It makes sense. I guess I was not looking at that way.